core/domain/role_domain_test.py

```diff
@@ -0,0 +1,134 @@
+# coding: utf-8
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""tests for role_domain"""
+
+import string
+
+from core.domain import role_domain
+from core.tests import test_utils
+
+
+class RoleDomainUnitTests(test_utils.GenericTestBase):
+    """test the role domain"""
+    HIERARCHY = role_domain.ROLE_HIERARCHY
+    ACTIONS = role_domain.ROLE_ACTIONS
+
+    def test_dicts_have_same_keys(self):
+        """test that ROLE_HIERARCHY and ROLE_ACTIONS have same keys
+        in same order.
+        """
+        self.assertEqual(self.HIERARCHY.keys(), self.ACTIONS.keys())
+
+    def test_dicts_have_list_value(self):
+        """test that ROLE_HIERARCHY and ROLE_ACTIONS, both have list as value
+        to all the keys
+        """
+        for i in self.HIERARCHY:
+            self.assertTrue(isinstance(self.HIERARCHY[i], list))
+
+        for i in self.ACTIONS:
+            self.assertTrue(isinstance(self.ACTIONS[i], list))
+
+    def test_valid_names(self):
```

Comment:
OK, I'm not really sure why we care about this... If we use named string constants then it doesn't really matter what the values of the constants are? The important thing is that they should not change.